src/enzyme_ad/jax/Implementations/HLODerivatives.td

```diff
@@ -603,6 +691,14 @@ def : HLODerivative<"MaxOp", (Op $x, $y),
                   (Select (Compare $x, $y, (LT)), (SelectIfActive $y, (Shadow $y), (HLOConstantFP<"0"> $y)), (SelectIfActive $x, (Shadow $x), (HLOConstantFP<"0"> $x)))
                   >;
 
+def : HLODerivative<"MinOp", (Op $x, $y),
+                  [
+                    (Select (Compare $x, $y, (GT)), (HLOConstantFP<"0"> $x), (DiffeRet)),
+                    (Select (Compare $x, $y, (GT)), (DiffeRet), (HLOConstantFP<"0"> $x))
```

Comment:
For improving optimization can we use LT for this too but swap lhs and rhs (that way min and max derivatives can be CSEd)